SteamKit2/SteamKit2/Steam/Handlers/SteamUnifiedMessages/Callbacks.cs

```diff
@@ -101,30 +99,26 @@ public string RpcName
             /// <summary>
             /// Gets the full name of the service method.
             /// </summary>
-            [DisallowNull, NotNull]
-            public string? MethodName { get; private set; }
+            public string MethodName { get; private set; }
 
             /// <summary>
             /// Gets the protobuf notification body.
             /// </summary>
-            [DisallowNull, NotNull]
-            public object? Body { get; private set; }
+            public object Body { get; private set; }
 
 
             internal ServiceMethodNotification( Type messageType, IPacketMsg packetMsg )
             {
                 // Bounce into generic-land.
                 var setupMethod = GetType().GetMethod( nameof(Setup), BindingFlags.Instance | BindingFlags.NonPublic ).MakeGenericMethod( messageType );
-                setupMethod.Invoke( this, new[] { packetMsg } );
+                (MethodName, Body) = ((string, object))setupMethod.Invoke( this, new[] { packetMsg } );
```

Comment:
This cast needs to be tested.

Reply:
I replaced the dll in steamdb and ran `pubfile` command which uses `UnifiedService<IPublishedFile>`, it worked. I am not 100% certain whether it went through this invocation though.

Reply:
IIRC I can trigger this by enabling parental controls. Let me check...

Reply:
Cast works but reflection returned null, lol